eng/scripts/Language-Settings.ps1

```diff
@@ -87,15 +87,21 @@ function Publish-javascript-GithubIODocs ($DocLocation, $PublicArtifactLocation)
   $PublishedDocs = Get-ChildItem "$($DocLocation)/documentation" | Where-Object -FilterScript { $_.Name.EndsWith(".zip") }
 
   foreach ($Item in $PublishedDocs) 
-  {
-    $PkgName = "azure-$($Item.BaseName)"
-    Write-Host $PkgName
+  {    
     Expand-Archive -Force -Path "$($DocLocation)/documentation/$($Item.Name)" -DestinationPath "$($DocLocation)/documentation/$($Item.BaseName)"
     $dirList = Get-ChildItem "$($DocLocation)/documentation/$($Item.BaseName)/$($Item.BaseName)" -Attributes Directory
 
     if ($dirList.Length -eq 1)
     {
       $DocVersion = $dirList[0].Name
+      $pkgs, $parsePkgInfoFn = RetrievePackages "NPM" $PublicArtifactLocation
```

Comment:
Why not call `Get-javascript-PackageInfoFromPackageFile` directly?

Reply:
Also the repository arguement is being retired from `RetrievePackages` function. So you should not need to pass NPN to the function.

Reply:
> Why not call `Get-javascript-PackageInfoFromPackageFile` directly?

I was trying to reuse Retrievepackages function to get package details from artifact path. But I makes sense to just call Get-ChildItem in Language-Settings itself to decouple from common method.